### ISSUE TITLE
Code Review

### ISSUE THREAD

**Issue by Saviour94** [MarcoCacoete/MCacoeteUOL#6]
Hey Marco,

From what I can tell the program is looking great.

1. As I'm not very proficient in C# it would be really beneficial if you could add some more comments into your code so that I can better understand what's going on and have an insight into your thought process.

2. Have you considered adding another class? This could help you chunk the problem a bit more and could help make your code more efficient.

3. I couldn't help but notice your code is sometimes hard to follow, if you improved the structure it would perhaps make the code more readable. 

Looking forward to seeing the end result!

**Comment by MarcoCacoete** [comment]
Thanks for the feedback, I've implemented some changes on my current built to better help inexperienced coders like us, I agree with your suggestion about classes, and it's in the works too.
